errortypes/severity.go

```diff
@@ -20,7 +20,7 @@ func isFatal(err error) bool {
 	return !ok || s.Severity() == SeverityFatal
 }
 
-func isWarning(err error) bool {
+func IsWarning(err error) bool {
```

Comment:
Since this is now being exported, might want to make a comment to explain that an error can only be a warning if it is a warning type from this package. There is no magic to divine that a random error is intended to only be a warning.

Reply:
Added a comment here and similar one in the definition of the `Warning` type. Let me know if you agree with the wording